python/fate_client/pipeline/component/README.rst

```diff
@@ -463,6 +463,14 @@ Below lists input and output elements of each component.
      - None
      - None
 
+   * - `Secret Sharing Sum`_
+     - Secret Sharing Sum
+     - This component will sum multiple privacy values without exposing data
+     - Table, values are addend
```

Comment:
here should be "data"; this document is on how to access input & output of pipeline components